core/document.adoc

```diff
@@ -0,0 +1,69 @@
+= OGC API - Processes - Part 1: Core
```

Comment:
@manuel489 let's not move the main document location, this creates lots of issues when merging.

Reply:
@ronaldtse what do you mean "main document"?
What I did was moving the sub-clause files to sections/ folder and rename the `18-062.adoc` file to `document.adoc`.

Reply:
Yes, I wanted to keep `18-062.adoc` where it was to ensure file changes are not too numerous. I've moved it back to `18-062.adoc` now, so it's all good. Thanks!